app/assets/javascripts/short-form/ShortFormApplicationService.js.coffee.erb

```diff
@@ -32,6 +32,7 @@ ShortFormApplicationService = (
       assistedHousingRentBurden: null
     householdIncome: { incomeTotal: null, incomeTimeframe: 'per_year' }
     groupedHouseholdAddresses: []
+    prioritiesSelected: {}
```

Comment:
lol can't believe we missed that in the 1000 test cases we tried! nice catch @kaptron